web/modules/custom/server_style_guide/src/Controller/StyleGuideController.php

```diff
@@ -936,4 +936,60 @@ protected function getWebformElement(): array {
     );
   }
 
+/**
+   * Person card demo page.
+   */
+  public function personCard() {
+  // Single card - center with constrained width (matches Figma).
+  $single = [
+    '#type' => 'container',
+    '#attributes' => ['class' => ['py-12', 'bg-gray-50', 'mb-8']],
+    'inner' => [
+      '#type' => 'container',
+      '#attributes' => ['class' => ['max-w-sm', 'mx-auto']],
+      'card' => [
+        '#theme' => 'person_card',
+        '#image' => 'https://randomuser.me/api/portraits/women/44.jpg',
+        '#name' => 'Jane Cooper',
+        '#role' => 'Paradigm Representative',
+        '#badge' => 'Admin',
+        '#email' => 'jane@example.com',
+        '#phone' => '+1234567890',
+      ],
+    ],
+  ];
+
+  // Grid of 10 cards (responsive).
+  $cards = [];
+  for ($i = 1; $i <= 10; $i++) {
+    $cards[] = [
+      '#theme' => 'person_card',
+      '#image' => 'https://randomuser.me/api/portraits/women/' . (10 + $i) . '.jpg',
+      '#name' => 'Person ' . $i,
+      '#role' => 'Team Member',
+      '#badge' => ($i % 3 === 0) ? 'Admin' : NULL,
+      '#email' => "person{$i}@example.com",
+      '#phone' => '+1234567890',
+    ];
+  }
+
+  $grid = [
+    '#type' => 'container',
```

Comment:
This isn't aligned with the rest of the codebase - we don't use CSS classes inside PHP